### DISCUSSION TITLE
Pytest JUnitXML-Report for  XRAY JIRA

### DISCUSSION THREAD

**Discussion by alex-scn** [pytest-dev/pytest#11629] [Q&A]
Hello,

i use the latest Pytest and want to set a test_id or test_key to link my tests with XRAY Jira. I found that i can use the [record_property fixture.](https://docs.pytest.org/en/7.1.x/reference/reference.html#record-property) 
When i use it i get the warning 
` record_property is incompatible with junit_family 'xunit2' (use 'legacy' or 'xunit1')` 
so my workaround is set to xunit1. I cant see informations in the docs that xunit2 is not supported. Will it be in the future or can i use another fixture/method to link test cases.

**Comment by RonnyPfannschmidt** [comment]
AS far as I understand, xunit2 style reports are not able to support the feature at all

So unless something is missing it's never gonna happen
